### PR TITLE
Create projects converter

### PR DESCRIPTION
**Pull request by StasGurin**

### CODE REVIEW

File: ConverterTest/Converter.cs [hunk 1]
```diff
@@ -154,12 +156,20 @@ public class Converter
 
                     foreach (var docProjects in dateProjects)
                     {
-                        var convertProject = new Project(docProjects.Id, docProjects.Type, docProjects.Name, docProjects.DomainName, docProjects.Keyword,
+                        var convertProject = new Project(docProjects.Id, docProjects.Type, docProjects.Name, docProjects.DomainName.Substring(7), docProjects.Keyword,
```
Comment (azakanych) [inline ConverterTest/Converter.cs]: why domain name is trimmed by 7?

**Review by azakanych — COMMENTED** [review]
1 Comment

File: ConverterTest/Converter.cs (hunk 1 above)
Comment (StasGurin) [reply ConverterTest/Converter.cs]: for delete 'http:\'

Comment (azakanych) [reply ConverterTest/Converter.cs]: To make code more readable better to use in such cases Replace("http:\", "")

**Pull request merged**